app/uk/gov/hmrc/timetopay/arrangement/services/TTPArrangementService.scala

```diff
@@ -80,11 +79,11 @@ class TTPArrangementService @Inject() (
                   AuditService.auditTags
                 )
 
-                returnedError
+                Future.failed(DesApiException(error.code, error.message, queuedForRetry = true))
```

Comment:
keeping with the spirit of the old code you could have done 
`def returnedError(queuedForRetry:Boolean) = Future.failed(DesApiException(error.code, error.message), queuedForRetry)`
and used  
`returnedError(queuedForRetry = true)`

(but more up to you than wrong/right)

Reply:
I like the spirit of old code. will change